caput/mpiarray.py

```diff
@@ -245,6 +237,92 @@ class MPIArray(np.ndarray):
         The dimension to distribute the array across.
     """
 
+    def __getitem__(self, v):
```

Comment:
Can you add a bit of documentation about what is and isn't supported in MPIArray? in terms of ufuncs and slicing etc, I think it would help people to have a clear description of that.

Reply:
Probably in the class level docstring.

Reply:
It doesn't need crazy detail. But something would be good. A few examples wouldn't hurt too, although you could just point people to the tests.

Reply:
Yes! I put these in some of the method docstrings, but I will write them in the class.